src/Terrain/MapMaster.java

```diff
@@ -66,7 +66,7 @@ public MapMaster(CommandingOfficers.Commander[] COs, MapInfo mapInfo)
             // If the CO has no HQ yet, assign this one.
             if( COs[co].HQLocation == null )
             {
-              System.out.println("Assigning HQ at " + x + ", " + y + " to " + COs[co]);
+//              System.out.println("Assigning HQ at " + x + ", " + y + " to " + COs[co]);
```

Comment:
Hmm.

Reply:
I figure we had it in there for some reason. I just got annoyed by seeing it spammed, lol